### PR TITLE
Connect with testing database

### PR DESCRIPTION
**Pull request by shahenazmonia**
* Add new heading to check which is database will be used, by adding new file has queries for testing database

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,20 @@ tests, run `npm run test` in your terminal.
   at its default state. That's why before running every single test we have to
   rerun the script from `db_build.js` to restart the database.
 
-### 4. Turn the db build script into a reusable function
+### 4. Connect with testing databas
+
+* A big projects maybe have different type of databas, database for development, database for production, and database for testing. To make sure you are running the testing databas, inside `db_build.js` we need to check that by:
```
Comment (SleepySheepy172) [inline README.md]: Just a few typos: first and last "database"  in line 121 are spelt "databas"

Comment (SleepySheepy172) [reply README.md]: There's another of the same on line 186 too

**Comment by SleepySheepy172** [comment]
@shahenazmonia Sorry to jump in but I'm mentoring week 6 in london so I've been going over the material in preparation and couldn't help but look over the changes being added

**Comment by mattlub** [comment]
@SleepySheepy172 Thanks for reviewing :) Me and @shahenazmonia will deliver this tomorrow in Khalil- probably from a fork/branch because it is quite late to merge changes- however since we are a day ahead of London you will be able to see our changes the night before you deliver it in London (if you want)

**Comment by mattlub** [comment]
closing this as we didn't actually use it 😕

**Pull request closed without merging**